### PR TITLE
Project documentation RustFlower

### PR DESCRIPTION
**Pull request by RaduRadel**
### Pull Request Overview

This pull request adds the documentation for the project called RustFlower and implements the necessary revisions.

### Testing Strategy

This pull request was tested by Baccela Radu-Costin.

### TODO or Help Wanted

This pull request still requires verification to ensure it meets the requirements.

### Build

- [ ] Ran `npm build`.

### Author

Signed-off-by: Baccela Radu-Costin <baccelaradu4@gmail.com>

### CODE REVIEW

File: docs/project/2024/Baccela Radu-Costin/index.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+The RustFlower project is designed to develop a solar tracker system that dynamically aligns a solar panel towards the brightest light source to maximize energy absorption. Using a set of photoresistors, the system detects light intensity from various angles and adjusts the panel's orientation accordingly. Additionally, the project incorporates an LCD display to provide real-time information on the amount of energy the solar panel produces. This functionality not only optimizes the energy efficiency of the solar panel but also serves as an educational tool to demonstrate the principles of photovoltaic energy generation and automated mechanical alignment.
+
+## Motivation
+
+I chose the solar tracker project after reviewing a variety of engaging projects from previous years. This project stood out to me not only because of its intrinsic interest but also because it presents a well-balanced challenge involving both hardware construction and software development.
+
+## Architecture 
+
+![Solar Tracker System Diagram](diagram.png "Diagram of the Solar Tracker System")
```
Comment (alexandruradovici) [inline docs/project/2024/Baccela Radu-Costin/index.md:22]: This syntax is incorrect, please try it with `npm start`.

File: docs/project/2024/Baccela Radu-Costin/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+| [cortex-m-rt](https://github.com/rust-embedded/cortex-m-rt) | Runtime for ARM Cortex-M microcontrollers. | Facilitates low-level device initialization and setup for real-time operations. |
+| [INA219](https://crates.io/crates/ina219) | Library for interfacing with the INA219 sensor over I2C. | Used to measure voltage, current, and calculate power production from the photovoltaic panel. |
+
+
+## Links
+
+1. [link](https://www.instructables.com/Solar-Tracker-Using-Arduino-and-Raspberry-Pi-3/)
+2. [link](https://github.com/polyideas/Raspberry-Pi-Solar-Tracker)
+3. [link](https://www.youtube.com/watch?v=_6QIutZfsFs&t=169s)
+4. [link](https://www.youtube.com/watch?v=lrugreN2K4w)
```
Comment (alexandruradovici) [inline docs/project/2024/Baccela Radu-Costin/index.md:104]: Please add the link titles.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill out the pull request template.

File: docs/project/2024/Baccela Radu-Costin/diagram.png
Comment (alexandruradovici) [inline docs/project/2024/Baccela Radu-Costin/diagram.png:1]: Rename the folder to your curs.upb.ro username.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
> Please fill out the pull request template.

This is  ot resolved.

**Comment by RaduRadel** [comment]
> > Please fill out the pull request template.
> 
> This is not resolved.

File: docs/project/2024/radu_costin.baccela/index.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+| [Female-Female Wires](https://www.optimusdigital.ro/ro/fire-fire-mufate/880-fire-colorate-mama-mama-10p-10-cm.html?search_query=Fire+Colorate+Mama-Mama+%2810p%29+10+cm&results=11) | Used for extending connections between components | [3 RON](https://www.optimusdigital.ro/ro/fire-fire-mufate/880-fire-colorate-mama-mama-10p-10-cm.html?search_query=Fire+Colorate+Mama-Mama+%2810p%29+10+cm&results=11) |
+| [Resistors](https://www.optimusdigital.ro/ro/componente-electronice-rezistoare/33-rezistoare-set.html?search_query=Set+de+Rezistoare+0.25W+20+de+valori+cate+20+buc.+fiecare&results=1) | Controls current flow and voltage levels | [15 RON](https://www.optimusdigital.ro/ro/componente-electronice-rezistoare/33-rezistoare-set.html?search_query=Set+de+Rezistoare+0.25W+20+de+valori+cate+20+buc.+fiecare&results=1) |
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```
Comment (alexandruradovici) [inline docs/project/2024/radu_costin.baccela/index.md:89]: Do you need this library?

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**